sql/aggregate/tract.sql

```diff
@@ -1,8 +1,8 @@
 DROP TABLE IF EXISTS AGGREGATE_tract_{{ decade }};
 SELECT 
     YEARLY_devdb_{{ decade }}.boro,
-    YEARLY_devdb_{{ decade }}.bct{{ decade }},
-    YEARLY_devdb_{{ decade }}.centract{{ decade }},
+    YEARLY_devdb_{{ decade }}.bct{{ decade }}::TEXT,
+    YEARLY_devdb_{{ decade }}.centract{{ decade }}::TEXT,
```

Comment:
this line introduces a duplicate column in the census tract aggregate output as discussed @td928

Reply:
@mbh329 new commit should fix this dup columns issue